src/braket/devices/local_simulator.py

```diff
@@ -261,66 +258,29 @@ def _(self, backend_impl: BraketSimulator):
         return backend_impl
 
     @singledispatchmethod
-    def _run_internal(
+    def _construct_payload(
         self,
-        task_specification: Union[
-            Circuit,
-            Problem,
-            OpenQASMProgram,
-            AnalogHamiltonianSimulation,
-            AHSProgram,
-            SerializableProgram,
-        ],
-        shots: Optional[int] = None,
-        *args,
-        **kwargs,
-    ) -> Union[GateModelQuantumTaskResult, AnnealingQuantumTaskResult]:
+        task_specification: Any,
```

Comment:
same question here, why don't we keep the more specific type hint for `task_specification`?